doc/source/reference/c-api.ufunc.rst

```diff
@@ -93,12 +93,18 @@ Functions
         the corresponding 1-d loop function in the func array.
 
     :param types:
-        Must be of length (*nin* + *nout*) \* *ntypes*, and it
-        contains the data-types (built-in only) that the corresponding
-        function in the *func* array can deal with.
+        Int8 of length `(nin + nout) * ntypes` It encodes the :ref:`dtype.num`
```

Comment:
`Int8` is only true on some platforms, where `char` is signed.


Might be better to refer to `PyArray_Descr.type_num` here, since that's the C-side name. Could refer to both.